netbox/dcim/models/cables.py

```diff
@@ -533,8 +534,8 @@ def from_origin(cls, terminations):
             return None
 
         # Ensure all originating terminations are attached to the same link
-        if len(terminations) > 1:
-            assert all(t.link == terminations[0].link for t in terminations[1:])
+        if len(terminations) > 1 and not all(t.link == terminations[0].link for t in terminations[1:]):
+            raise UnsupportedCablePath(_("All originating terminations must start must be attached to the same link"))
```

Comment:
Please review the message content, the sentence is incorrect in its current state

Reply:
Fixed this, thanks for the catch